control/init.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash
+if ! hash virtualenvwrapper.sh 2>/dev/null; then
+    echo "Please install virtualenvwrapper and make sure it is in your PATH"
+    echo ""
+    echo "  sudo apt-get install git python-dev python-pip"
+    echo "  sudo pip install virtualenv virtualenvwrapper"
+    return 1
+fi
+
+source virtualenvwrapper.sh
+
+if [ ! -d ~/.virtualenvs/ansible ]; then
+    echo "Creating ansible virtualenv..."
+    mkvirtualenv ansible
+    pip install -r ~/commcarehq-ansible/ansible/requirements.txt
+else
+    workon ansible
+fi
+
+if ! [ -L ~/commcarehq-ansible/config -o -d ~/commcarehq-ansible/config ]; then
+    # TODO setup permissions (add user to group?) to allow write to config repo
+    #su ansible -c "usermod -a -G ansible $USER"
+    ln -s /home/ansible/commcarehq-ansible/config ~/commcarehq-ansible/config
```

Comment:
Everyone's voting for the same option so I'm going to implement the ❤️. This will entail a small adjustment to our current process: changes in the config repo in the ansible account will also need to be pushed to the central repo while the ansible account is still in use. Feel free to point out if I've overlooked some issue concerning security of our secrets. I think the security is unchanged, but could have overlooked something.